src/utils/functionRuntimeUtils.ts

```diff
@@ -52,23 +52,33 @@ export namespace functionRuntimeUtils {
                             'Your version of the Azure Functions Core Tools ({0}) does not match the latest ({1}). Please update for the best experience.',
                             localVersion,
                             newestVersion);
+                        const v2: string = localize('v2BreakingChanges', 'Caution: As v2 is still in preview, updating could result in breaking changes.');
```

Comment:
I'm struggling with this message - the hard part is that we want it to make them _more_ likely to update rather than less likely (since their Azure function app always gets updated with the latest and may be in a broken state).

Again, here's the shortest message I've come up with:
> v2 is in preview and may have breaking changes (which are applied automatically in Azure).

Reply:
This means that their Azure Function app will always be updated to the most current release?

Reply:
Yep it will